api2/go.mod

```diff
@@ -3,24 +3,23 @@ module github.com/nrc-no/core-kafka
 go 1.16
```

Comment:
Why are there changes to go.mod ?

Reply:
When I ran it in Goland it automatically changed go.mod

Reply:
Ran `go mod tidy` as suggested by @ludydoo on slack